src/control/lib/control/system.go

```diff
@@ -566,41 +566,49 @@ func SystemExclude(ctx context.Context, rpcClient UnaryInvoker, req *SystemExclu
 	return resp, convertMSResponse(ur, resp)
 }
 
+// SystemOsaResult describes the result of an OSA operation on a pool's ranks.
```

Comment:
As noted elsewhere, this is not a very human-friendly type name. It requires the reader to go figure out what `Osa` or `OSA` is. That's more of an internal DAOS name, IMO. Better to describe the purpose of the types and functions in terms of why and how they would be used, e.g. `SystemPoolMembershipUpdate` or something along those lines. Yes, it's more verbose, but this is a public API, and I don't think brevity is a virtue in public types and documentation.

Reply:
Membership is an overloaded term IMO but if you insist I can change to SystemPoolMembershipUpdate, is that required?

Reply:
Put yourself in the shoes of an API user who has not been hacking on DAOS for a couple of years. What the heck is a/an `Osa`? Sure, it can be looked up, but the bar for adding to an API surface should be higher than "it compiles and lets me move on to the next bug, go figure it out", right?

Reply:
I'm re-factoring to reduce duplication as per suggestion from @wangshilong and will rename appropriately